openstack_dashboard/api/rest/nova.py

```diff
@@ -360,12 +360,20 @@ def post(self, request):
 
         This returns the new server object on success.
         """
+        # Since baremetal sites only have one flavor, just grab the ID from the first one
+        try:
+            flavors = api.nova.flavor_list(request)
+            flavor_id = flavors[0].id
```

Comment:
it would be better to get the flavor with the name "baremetal" if possible. Otherwise this could break if an extra flavor is added for some reason.

Reply:
Done.